packages/Icon/src/Ellipsis.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+
+const SvgEllipsis = ({ title, ...props }) => (
+  <svg
+    width="1em"
+    height="1em"
+    viewBox="0 0 24 24"
+    css={`
+      color: ${props.color};
+      width: ${props.size};
+      height: ${props.size};
+      vertical-align: text-top;
+    `}
+    focusable={false}
+    {...props}
+  >
+    <title>Ellipsis</title>
```

Comment:
The other icons have `<title>{title}</title>`. Not sure how important that is

Reply:
Yup me neither but I wasn't about to break convention...

Reply:
But you did break convention. The other icons have <title>{title}</title>.

Reply:
i see, ok replaced with `title`